tensorboard/components/tf_paginated_view/tf-paginated-view2.html

```diff
@@ -256,16 +256,30 @@
 
     behaviors: [
       tf_dashboard_common.ArrayUpdateHelper,
-      Polymer.Templatizer,
     ],
 
-    ready() {
-      // Template instance props that should be excluded from forwarding
-     this._instanceProps = {
-       [this.as]: true,
-     };
-
-      this.templatize(this.$.template.querySelector('template'));
+    _ensureTemplatized() {
+      // Polymer is not ready (and props/DOM for the components are not
+      // populated)
+      if (!this.isAttached) return false;
```

Comment:
If `isAttached` is deprecated (as suggested in the PR description), do
we want to use an alternative here? Like, manage an `_attached` property
ourselves?

(I also can’t find docs for `isAttached` anywhere: neither the [API docs
for `Polymer.Base`][base] nor the [“Registration and lifecycle”
guide][ral]. What do you use for a reference?)

[base]: https://polymer-library.polymer-project.org/1.0/api/classes/Polymer.Base
[ral]: https://polymer-library.polymer-project.org/1.0/docs/devguide/registering-elements

Reply:
I find it super odd that `isAttached` is not present in the doc and this PR does not seem to do anything. https://github.com/Polymer/polymer/pull/4368/files but is present in code:
1. https://github.com/Polymer/polymer/blob/2.x/lib/legacy/legacy-element-mixin.html#L112
2. https://github.com/Polymer/polymer/blob/2.x/types/lib/legacy/legacy-element-mixin.d.ts#L40

Although `isAttached` has very different meaning in Polymer2, the prop is _not_ removed/deprecated.